content/docs/pulumi-cloud/deployments/using/settings.md

```diff
@@ -53,6 +53,20 @@ When using Pulumi Deployments, you have options for where your deployments run:
 
 For more information on customer-managed agents, see the [Customer Managed Agents documentation](../../customer-managed-agents).
 
+### Role assignment
+
+When configuring deployment settings, you can assign organization roles to the stack token used for deployments. This setting appears as a dropdown menu under "Role assignment" that displays available organization roles.
+
+If no role is selected, the deployment will only have access to the specific stack being deployed. However, this limited access can cause failures when the deployment needs to:
+
+- Access stack references from other stacks
+- Access environments
+- Manage organization resources such as teams, members, or OIDC issuers
+
+By selecting an appropriate role, you provide the deployment with the necessary permissions to access these additional resources. For fine-grained access control, you can create custom roles with specific permissions tailored to what the deployment needs to accomplish.
```

Comment:
If they don't have access to custom roles (i.e. on a team plan), they can still assign the default admin/member roles, right?

Reply:
mm good question, I think so. We should check for the org setting here tho https://github.com/pulumi/pulumi-service/blob/9f18ccce998c6c154147e8c529f49eae601e90a1/cmd/console2/src/app/pages/stack/settings/deploy/deploy-settings.component.ts#L197

instead of just the flag